include/zephyr/bluetooth/cs.h

```diff
@@ -540,6 +590,35 @@ struct bt_le_cs_subevent_step {
 	const uint8_t *data;
 };
 
+/** Sign-extended IQ sample extracted from step data. */
+struct bt_le_cs_iq_sample {
+	int16_t i;
+	int16_t q;
+};
+
+/** @brief Extract IQ sample from HCI-formatted PCT.
+ *
+ * Convenience function for processing 24-bit phase correction terms found
+ * in CS step data. The 12-bit signed real and imaginary components are
+ * converted to host endianness and sign-extended.
+ *
+ * @param pct 24-bit little-endian phase correction term.
+ *
+ * @return Real and imaginary terms as int16_t.
```

Comment:
It's not returning `int16_t` though, rather `struct bt_le_cs_iq_sample`.

Reply:
> It's not returning `int16_t` though, rather `struct bt_le_cs_iq_sample`.

Nevermind - it's just a reference to the implementation of `structure bt_le_cs_iq_sample`. It's just not clear from the text alone that it contains two `int16_t` parts.

Reply:
can rephrase to `@return bt_le_cs_iq_sample struct containing real and imaginary terms as int16_t`

Reply:
Sure